site/content/3.11/develop/drivers/java/_index.md

```diff
@@ -59,6 +64,287 @@ dependencies {
+System.out.println("Attribute c: " + updatedDocument.getAttribute("c"));
+```
+
+After executing this program, the console output should look like this:
+
+```text
+Key: myKey
+Attribute a: Foo
+Attribute b: 42
+Attribute c: Bar
+```
+
+### Delete a document
+
+Let's delete a document:
+
+```java
+System.out.println("Deleting document ...");
+collection.deleteDocument(key);
+```
+
+### Execute AQL queries
+
+First, you need to create some documents with the name `Homer` in the
+collection called `firstCollection`:
+
+```java
+for (int i = 0; i < 10; i++) {
+    BaseDocument value = new BaseDocument(String.valueOf(i));
+    value.addAttribute("name", "Homer");
+    collection.insertDocument(value);
+}
+```
+
+Get all documents with the name `Homer` from the collection using an AQL query
+and iterate over the results:
+
+```java
+String query = "FOR t IN firstCollection FILTER t.name == @name RETURN t";
+Map<String, Object> bindVars = Collections.singletonMap("name", "Homer");
+System.out.println("Executing read query ...");
+ArangoCursor<BaseDocument> cursor = db.query(query, bindVars, null, BaseDocument.class);
+cursor.forEach(aDocument -> System.out.println("Key: " + aDocument.getKey()));
+```
+
+After executing this program, the console output should look something like this:
+
+```text
+Key: 1
+Key: 0
+Key: 5
+Key: 3
+Key: 4
+Key: 9
+Key: 2
+Key: 7
+Key: 8
+Key: 6
+```
+
+Some details you should know about the code:
+
+- The AQL query uses the placeholder `@name` that has to be bound to a value
+- The `query()` method executes the defined query and returns an `ArangoCursor`
+  with the given class (here: `BaseDocument`)
+- The order of is not guaranteed
+
+### Delete documents with AQL
+
+Delete previously created documents:
+
+```java
+String query = "FOR t IN firstCollection FILTER t.name == @name "
+    + "REMOVE t IN firstCollection LET removed = OLD RETURN removed";
+Map<String, Object> bindVars = Collections.singletonMap("name", "Homer");
+System.out.println("Executing delete query ...");
+ArangoCursor<BaseDocument> cursor = db.query(query, bindVars, null, BaseDocument.class);
+cursor.forEach(aDocument -> System.out.println("Removed document " + aDocument.getKey()));
+```
+
+After executing this program, the console output should look something like this:
+
+```text
+Removed document: 1
+Removed document: 0
+Removed document: 5
+Removed document: 3
+Removed document: 4
+Removed document: 9
+Removed document: 2
+Removed document: 7
+Removed document: 8
+Removed document: 6
+```
+
+### Learn more
+
+- Have a look at the [AQL documentation](../../../aql/) to lear about the
+  query language
+- See [Serialization](reference-version-7/serialization.md) for details about
+  user-data serde
+- For the full reference documentation, see
+  [JavaDoc](https://www.javadoc.io/doc/com.arangodb/arangodb-java-driver/latest/index.html)
+
 ## GraalVM Native Image
 
 The driver supports GraalVM Native Image compilation.
```

Comment:
Same issue with the link here https://github.com/arangodb/arangodb-java-driver/tree/main/driver/src/test/java/graal